pyproject.toml

```diff
@@ -59,9 +60,8 @@ keyring = ["keyring"]
 template = [
     "pdm[copier,cookiecutter]",
 ]
-truststore = ["truststore; python_version >= \"3.10\""]
```

Comment:
I don't think the `[truststore]` extra was used much so this shouldn't be a huge problem (likely many people were getting truststore via `pdm[all]`) so this should be fine? The `pdm[truststore]` extra isn't in the docs, but there is a separate section on installing `truststore` which may not be needed anymore now that it's a default?

Reply:
Requesting for non-existing extra names will result in a warning. So that is okay.